odra-vm/src/vm/utils.rs

```diff
@@ -1,7 +1,5 @@
-use odra_core::{
-    casper_types::{account::AccountHash, ContractPackageHash},
-    Address
-};
+use odra_core::casper_types::PackageHash;
+use odra_core::{casper_types::account::AccountHash, Address};
 
 pub fn account_address_from_str(str: &str) -> Address {
```

Comment:
Could this method live in `address.rs`?

Reply:
This code is used only by odra_vm, mostly for tests, so there is no point of littering address.